e2e/test/scenarios/filters-reproductions/filters-reproductions.cy.spec.js

```diff
@@ -1482,3 +1482,41 @@ describe("issue 47887", () => {
     });
   });
 });
+
+describe("Issue 48851", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsNormalUser();
+  });
+
+  const manyValues = Array(75)
```

Comment:
I think we need a few more values. With 75 the "Add filter" button is still visible in the viewport ([image](https://github.com/user-attachments/assets/c81f9627-5fb9-4199-aecb-c8e0f71ea63f)). 

Or maybe just use smaller viewport for this test.

Reply:
The test breaks for me locally without the fix. Assuming the viewport is the same for tests on CI, it should break there as well, right?

<img width="1624" alt="Screenshot 2024-10-28 at 22 09 35" src="https://github.com/user-attachments/assets/485a7854-33b5-4d52-8aa7-46e46fd3a7f6">

That being said, I've reduced the viewport size for the test to make it faster.